### PR TITLE
INT-4495 - Adding lastLogin property from rawData

### PR DESCRIPTION
**Pull request by adam-in-ict**

### CODE REVIEW

File: src/steps/access.ts [hunk 1]
```diff
@@ -85,6 +85,7 @@ export async function fetchUsers({
           mobilePhone: employee?.mobilePhone,
           workPhone: employee?.workPhone,
           supervisor: employee?.supervisor,
+          lastLogin: parseTimePropertyValue(user.lastLogin),
```
Comment (VDubber) [inline src/steps/access.ts:88]: Any unit tests around this?

**Review by VDubber — COMMENTED** [review]
What about gitleaks.toml?

File: src/steps/access.ts (hunk 1 above)
Comment (adam-in-ict) [reply src/steps/access.ts:88]: Whoops.  Didn't get my test update pushed.  Should be there now.  Thanks!

I didn't add in any new tests, just updated the existing to add this property.

**Comment by adam-in-ict** [comment]
> What about gitleaks.toml?

Thanks! Totally spaced that one.

**Review by VDubber — APPROVED** [review]

**Pull request merged**